packages/curvenote-cli/src/submissions/utils.ts

```diff
@@ -82,7 +80,7 @@ export async function postNewWork(
   cdnKey: string,
   cdn: string,
   key: string,
-): Promise<LogInfo> {
+): Promise<WorkDTO> {
```

Comment:
These functions which create works / work versions / submissions / submission versions now simply return the newly created object, rather than just the `LogInfo` (it's easier to build the log info in the containing functions where we pull the different parts together).